example/MyFirstDrawing/MyFirstDrawingUITests/BrushToolView/GroupPencilSizeViewUITests.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  GroupPencilSizeViewUITests.swift
+//  MyFirstDrawingUITests
+//
+//  Created by Stant Macmini n04 on 08/07/19.
+//  Copyright © 2019 Stant 02. All rights reserved.
+//
+
+import XCTest
+import StantDrawing
+@testable import MyFirstDrawing
+
+class GroupPencilSizeViewUITests: XCTestCase {
+    let app = XCUIApplication()
+    
+    override func setUp() {
+        continueAfterFailure = false
+
+        app.launch()
+        
+        let btnPencil = app.buttons["pencilIdentifier"]
+        btnPencil.tap()
+    }
+    
+    override func tearDown() {}
+    
+    func testClickOnGroupPencilSizeViewAndShowAllButtons() {
+        let pencilSizeReference = app.images.matching(identifier: "pencilSizeReferenceIdentifier")
+        
+        let btnPencilSize1 = app.buttons["pencilSize1Identifier"]
+        let btnPencilSize2 = app.buttons["pencilSize2Identifier"]
+        let btnPencilSize3 = app.buttons["pencilSize3Identifier"]
```

Comment:
Same comment above. If we decided to change over there, change it here as well.

Reply:
Done